pkg/awsutil/session.go

```diff
@@ -43,28 +48,56 @@ func (c *Credentials) Validate() error {
 }
 
 func (c *Credentials) NewSession(region string) (*session.Session, error) {
-	if c.HasProfile() {
-		return Log(session.NewSessionWithOptions(session.Options{
+	var opts session.Options
+
+	global := false
+
+	if region == GlobalRegionID {
+		region = DefaultRegionID
+		global = true
+	}
+
+	switch {
+	case c.HasProfile() && c.HasKeys():
```

Comment:
Instead of throwing an error of neither are present you now only throw one if both are present. Is this intended?

Reply:
No. Fixed.